pkg/extension/templatesapp/README.md

```diff
@@ -0,0 +1,8 @@
+```bash
+helm upgrade --install ingress-nginx ingress-nginx \
+  --repo https://kubernetes.github.io/ingress-nginx \
+  --namespace ingress-nginx --create-namespace \
+  --set controller.service.type=NodePort \
+  --set controller.service.nodePorts.http=30888
+```
```

Comment:
What are these? They seem irrelevant to the contents of this directory, and this is inconsistent with the content in `README_zh`.

Reply:
Done